pkg/output/table_test.go

```diff
@@ -261,8 +261,8 @@ var _ = Describe("Table", func() {
 		// Check the generated text:
 		lines := strings.Split(buffer.String(), "\n")
 		Expect(lines).To(HaveLen(3))
-		Expect(lines[0]).To(Equal(`ID                                NAME                        `))
-		Expect(lines[1]).To(Equal(`123                               mycluster                   `))
+		Expect(lines[0]).To(Equal(`ID                                NAME                                                  `))
+		Expect(lines[1]).To(Equal(`123                               mycluster                                             `))
```

Comment:
What is the change here, just whitespace?

Reply:
ahh the GitHub diff did not show the trailing whitespaces
Could we use some string repeat of space?

Reply:
To reflect the output of the table after the increase in width to 54 chars.

Reply:
We could use string repeat of space but I don’t see how that will improve it; I mean visually looking at the output like this this helps because it matches what really is display when the list cmd is run something that we won’t have with string repeat.

Reply:
I was mainly thinking about easy updating of the tests as I don't want to count the exact number of whitespace

or additionally, the issue I had with GitHub ignoring trailing whitespace 
![image](https://github.com/openshift-online/ocm-cli/assets/18332586/466c57dc-b217-4542-8c4e-54684b179f11)

Reply:
I would prefer to keep it like this for readability: with just a quick look it is clear what we are expecting to be inside the `lines`

@mnecas to me github shows clearly the trailing whitespace:
![image](https://github.com/openshift-online/ocm-cli/assets/1125019/de2347fa-e7cb-443e-ac0c-cdd265198c13)

Reply:
okay, lgmt

Reply:
Thanks for clarifying your point, which I see. I think it's a tradeoff between visually seeing the expected output vs speed of test update. I personally prefer the former for expected output like this that needs visual inspection.